tests/test_TFNetworkLayer.py

```diff
@@ -920,7 +920,8 @@ def test_CombineLayer_broadcast():
     net_dict = {
       "lin1": {"class": "linear", "activation": "sigmoid", "n_out": 5, "from": "data:data"},
       "lin2": {"class": "linear", "activation": "sigmoid", "n_out": 1, "from": "data:data"},
-      "combine": {"class": "combine", "kind": "add", "from": ["lin1", "lin2"]},
+      "lin2_squeeze": {"class": "squeeze", "from": "lin2", "axis": "f"},
```

Comment:
I wonder whether we should somehow simplify this, and unify into a single layer. But not sure. Maybe not.

Reply:
True. It is weird to think of `ExpandDimsLayer` as deprecated, but at the same time have valid use cases for `SqueezeLayer`.

Maybe via a special `out_dim` or so. But not too important.